client/src/pages/Restaurant/RestaurantView/index.js

```diff
@@ -3,21 +3,52 @@ import FoodList from "../FoodList";
 import Header from "./Header";
 import RestaurantInfo from "./RestaurantInfo";
 import { useState } from "react";
+import InputLabel from "@mui/material/InputLabel";
+import MenuItem from "@mui/material/MenuItem";
+import FormControl from "@mui/material/FormControl";
+import Select from "@mui/material/Select";
 
 export default function RestaurantView({ restaurant, foods }) {
   const [budget, setBudget] = useState("");
-  const foodsInBudget = foods.filter((food) => {
+  const [sortDirection, setSortDirection] = useState("");
+
+  const sortedBudgetFoods = foods.sort((a, b) => {
+    return sortDirection === -1 ? b.price - a.price : a.price - b.price;
+  });
+
+  const foodsInBudget = sortedBudgetFoods.filter((food) => {
     if (budget === "") {
       return true;
     }
     return food.price <= budget;
   });
 
+  const handleSortChange = (event) => {
+    setSortDirection(event.target.value);
+  };
+
   return (
     <div style={{ display: "flex", flexDirection: "column", gap: "0.5rem" }}>
       <Header restaurant={restaurant} />
       <RestaurantInfo restaurant={restaurant} />
-      <BudgetFilter foodsInBudget={foodsInBudget} budget={budget} setBudget={setBudget}/>
+      <BudgetFilter
+        foodsInBudget={foodsInBudget}
+        budget={budget}
+        setBudget={setBudget}
+      />
+      <FormControl sx={{ m: 1, maxWidth: 140, minWidth: 100 }}>
+        <InputLabel id="demo-simple-select-helper-label">Price Sort</InputLabel>
+        <Select
+          labelId="food-sorter"
+          id="food Sorter"
+          value={sortDirection}
+          label="foodOrder"
+          onChange={handleSortChange}
```

Comment:
if you really to use select, use the "standard" variant. See how I did our restaurant sorts